<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDS Header Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="hLabel">
       <property name="text">
        <string>Высота</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hInput">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>131072</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="wLabel">
       <property name="text">
        <string>Ширина</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="wInput">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>131072</number>
       </property>
       <property name="value">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="cLabel">
       <property name="text">
        <string>Кодек</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cInput"/>
     </item>
     <item>
      <widget class="QLabel" name="oLabel">
       <property name="text">
        <string>Смещение</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="oData">
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="fileBTN">
       <property name="text">
        <string>Выбрать файл </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="okBTN">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="folderBTN">
       <property name="text">
        <string>Выбрать папку</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="cancelBTN">
       <property name="text">
        <string>OTMEHA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
